src/UNL/Peoplefinder/Driver/LDAP.php

```diff
@@ -99,6 +100,9 @@ class UNL_Peoplefinder_Driver_LDAP implements UNL_Peoplefinder_DriverInterface
 
     public function __construct()
     {
+        $this->cache = UNL_Peoplefinder_Cache::factory([
```

Comment:
I'd rather this be a lazy loaded member ala `getCache()`. As we can see from the tests, the sample config is currently constructing this object, even when it will not be used.